tools/cdk-build-tools/lib/compile.ts

```diff
@@ -22,6 +22,7 @@ export async function compileCurrentPackage(timers: Timers, options: CDKBuildOpt
     '.',
     '--ext=.js,.ts',
     '--ignore-path=.gitignore',
+    `--resolve-plugins-relative-to=${__dirname}`,
```

Comment:
That'll only work "by luck" because everything is hoisted to the root. Most of the linters are, in fact, installed by dependency of `cdk-build-tools` and expected to resolve relative to there.

Reply:
`__dirname` is `cdk-build-tools/lib` which will resolve to the deps of `cdk-build-tools`, no?

Reply:
Oh I somehow though about it in the other way around. The actual problem at hand is caused (or maybe also caused) by missing `--no-eslintrc` here actually.